hexanenetlibrarydebug.lua

```diff
@@ -4,6 +4,14 @@
 
 hexanenetlibrarydebug = hexanenetlibrarydebug or {}
 
+hexanenetlibrarydebug.antiNetSpam = {}
+hexanenetlibrarydebug.flaggedPlayers = {}
+
+timer.Create("CleanNetSpam", 1,0, function()
```

Comment:
Seems like there might be a better way to do the garbage collection.
I guess it will be a bit worse to add expiry times to the *log* and have them decaying

Reply:
This is the best way, even spawning a mass of coroutines would cause more performance implications.